Dockerfile

```diff
@@ -0,0 +1,31 @@
+# use bullseye variant because it doesn't have network issue in m1 mac & intel
+FROM php:7.4-fpm-bullseye
+
+# Install dependencies
+RUN apt-get update
+RUN apt-get install -y zip nginx git
+RUN useradd -r nginx
+
+# Install PHP-Mysql extensions
+RUN docker-php-ext-install pdo pdo_mysql \
+    && docker-php-ext-enable pdo_mysql
+
+WORKDIR /var/www
+
+# Copy source code
+COPY . .
+RUN mv nginx.conf /etc/nginx/nginx.conf
+
+# Install composer
+COPY --from=composer:2.2 /usr/bin/composer /usr/bin/composer
+
+RUN composer install
+RUN composer dumpautoload --optimize
+
+# Copy .env file
+COPY .env.example .env
```

Comment:
I am getting this warning when I tried to run `docker-compose up --build -d`
```
$ docker-compose up --build 
WARN[0000] The "PUSHER_APP_CLUSTER" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PUSHER_APP_CLUSTER" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PUSHER_APP_CLUSTER" variable is not set. Defaulting to a blank string. 
```

Do you have any idea why this is happening?

![screen_2024-05-13_002](https://github.com/nafiesl/silsilah/assets/8721551/62096006-c56d-444d-9b63-1ff50dabc1d2)

Reply:
Thats because we dont set on this [env](https://github.com/nafiesl/silsilah/blob/master/.env.example#L37-L38) or in the local system environment.
Since we are not using it on this PR, I suggest to remove it.

Also I also thinking to remove all unnecessary env config on the `.env.example` and keep the required one. The reasons are to make the app clean and we dont need to think to add it when we want to deploy it

What do you think? @nafiesl

Reply:
@ilhamsyahids, OK thanks for the explanation, I found that the warning message is showing because my `.env` file in my localhost contains those 2 variables (which are not being used anywhere).

After I removed those 2 lines for my .env variable, the warning was gone.
```
MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
```

So I think this is not an issue anymore, I will approve and merge this PR. Thanks.